crates/cli/src/meta/mod.rs

```diff
@@ -407,6 +410,53 @@ pub async fn search_deployer(
     Ok(response_value)
 }
 
+/// checks if the given contract implements ERC165
+/// the process is done as described per ERC165 specs:
+/// https://eips.ethereum.org/EIPS/eip-165#how-to-detect-if-a-contract-implements-erc-165
+pub async fn supports_erc165(client: &ReadableClientHttp, contract_address: Address) -> bool {
```

Comment:
@rouzwelt i still think this is in the wrong place

@thedavidmeister what do you think? a new repo for this 1 fn seems like overkill but i can't think of another crate that it should be in

Reply:
yea my thought process was exactly the same as yours @hardyjosh , so that why I ended up putting it here for now thinking we can replace it later if we manage to need it across repos and elsewhere, but atm this is the only place we need this functionality, so let me know how it goes

Reply:
@rouzwelt @hardyjosh why not a new repo?

small repos are great, they're easy to stabilise

Reply:
@hardyjosh @thedavidmeister should we have some sort of a `util` repo? like `rain.utils`, I know utils are not something we do, but maybe for some of these small functionalities that are unrelated to eachother most of the times that we need across repos, it makes sense

Reply:
@rouzwelt let's just maka new repo, rain.interface, with a new crate with this fn. Then we can close this off.

Reply:
@rouzwelt repo should be rain.erc, will be a crate for ERC helpers.

Per convo with @thedavidmeister in tg

Reply:
it is moved to: https://github.com/rainlanguage/rain.interface